lib/OpenQA/Schema/Result/JobNextPrevious.pm

```diff
@@ -36,7 +36,7 @@ __PACKAGE__->result_source_instance->view_definition(
     WITH allofjobs AS(
     SELECT me.*
     FROM jobs me WHERE me.state=?
-    AND me.result NOT IN (?, ?, ?, ?, ?, ?, ?)
+    AND me.result NOT IN (?, ?, ?, ?, ?, ?, ?, ?)
```

Comment:
Could we use named references here? This array of question marks is rather opaque and relies on being called correctly from elsewhere.

Reply:
Maybe - I haven't done this yet with DBIx.